src/SubscribeMe/Interface/SubscriberInterface.php

```diff
@@ -2,9 +2,10 @@
 
 declare(strict_types=1);
 
-namespace SubscribeMe\Subscriber;
+namespace SubscribeMe\Interface;
```

Comment:
```suggestion
namespace SubscribeMe\Subscriber;
```

I think it's better to leave the `interface` among its implementation classes. It's better for this library legibility.